src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" method="get"
+      data-qa="big">
+      <label class="Search">
+        <img class="search-image"
+           src="images/Search.svg"
+           alt="Search"
+        >
```

Comment:
```suggestion
        <img
          class="search-image"
          src="images/Search.svg"
          alt="Search"
        >
```
here as well - fix it in all places